### PR TITLE
chore: added rock people and rock tags dags for City First and Oak Cliff

### PR DESCRIPTION
**Pull request by nlewis84**

### CODE REVIEW

File: dags/churches/city_first/city-first-rock-people-dag.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from airflow import DAG  # noqa: F401
+
+from datetime import datetime, timedelta
+
+from rock.rock_people_dag import create_rock_people_dag
```
Comment (vinnyjth) [inline dags/churches/city_first/city-first-rock-people-dag.py:5]: this import is wrong I think :)

**Comment by nlewis84** [comment]
@vinnyjth I wasn't totally sure on the best way to customize the rock_tag_dag, so I just made a new one that is specific for only people_tags. I am getting a None error currently that I'm not sure about, but may be related to the fact that we do not have a Rock Token yet, and I just defined that variable as an empty string.

**Comment by vinnyjth** [comment]
@nlewis84 Looks great to me. Once we get the Rock tokens I'll test and merge this.

**Comment by vinnyjth** [comment]
@nlewis84 let me know when we have rock variables for both churches

**Review by vinnyjth — APPROVED** [review]
💥

**Pull request merged**